### PR TITLE
Template statistics now surfaced on dashboard.

### PR DESCRIPTION
**Pull request by ashimali**
Job list removed.

Template statistics retrieved at same time as
notification stats.

### CODE REVIEW

**Comment by quis** [comment]
Should have tests for the client new API client, eg: https://github.com/alphagov/notifications-admin/blob/master/tests/app/main/notify_client/test_notification_client.py

**Comment by ashimali** [comment]
@quis test added

File: app/main/views/dashboard.py [hunk 1]
```diff
@@ -36,7 +45,8 @@ def service_dashboard(service_id):
         spent_this_month='0.00',
         statistics=add_rates_to(statistics),
         templates=templates,
-        service_id=str(service_id))
+        service_id=str(service_id),
```
Comment (quis) [inline app/main/views/dashboard.py]: You don’t need to pass this through any more, do `current_service.id` in the template instead.

File: app/templates/views/dashboard/template-statistics.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{% from "components/table.html" import list_table, field, right_aligned_field_heading %}
+{% call(item) list_table(
+  template_statistics,
+  caption="Recent templates used",
+  empty_message='You haven’t sent any batch messages yet',
+  field_headings=['Template', 'Type', right_aligned_field_heading('Usage')]
+) %}
+  {% call field() %}
+    <a href="{{ url_for('.edit_service_template', service_id=service_id, template_id=item.template.id) }}">
```
Comment (quis) [inline app/templates/views/dashboard/template-statistics.html]: `service_id` is missing here when this is rendered from the `.json` endpoint.

**Comment by servingUpAces** [comment]
:+1:

**Comment by quis** [comment]
:+1: :+1:

**Pull request merged**